src/pages/index.vue

```diff
@@ -97,6 +99,10 @@ function updatePoolFilters(feature: PoolTypeFilter | undefined) {
       filterPoolIds.value = [];
       filterPoolTypes.value = [PoolType.LiquidityBootstrapping];
       break;
+    case PoolTypeFilter.LRT:
+      filterPoolIds.value = lrtPools;
```

Comment:
hm, does this work? shouldn't we pass just ids there?

Reply:
Should have been `= lrtPools.value`